pkgs/applications/video/webtorrent_desktop/default.nix

```diff
@@ -33,7 +37,7 @@ runCommand "${pname}-${version}" rec {
       fontconfig freetype gdk-pixbuf glib gtk3 pango libuuid libX11
       libXScrnSaver libXcomposite libXcursor libXdamage libXext
       libXfixes libXi libXrandr libXrender libXtst libxcb nspr nss
-      stdenv.cc.cc udev
+      stdenv.cc.cc udev libdrm mesa
```

Comment:
I think this also needs the gappswrapper.

I get this when trying to get a file dialog:

```
[nix-shell:~/.cache/nixpkgs-review/pr-206051]$ WebTorrent 
init: 503.163ms
Xlib: sequence lost (0x10202 > 0x203) in reply type 0xc!
Xlib: sequence lost (0x10e4b > 0xe4f) in reply type 0x23!
Cannot upload crash dump: cannot exec /usr/bin/wget

Unexpected crash report id length
Failed to get crash dump id.
Report Id: 

(WebTorrent:2): GLib-GIO-ERROR **: 22:00:03.088: Settings schema 'org.gtk.Settings.FileChooser' is not installed
Cannot upload crash dump: cannot exec /usr/bin/wget

Unexpected crash report id length
Failed to get crash dump id.
Report Id: 
```

It's a bit tricky with buildFHSEnvUserEnv but maybe you find a way?